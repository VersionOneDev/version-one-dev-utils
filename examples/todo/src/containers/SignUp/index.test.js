// // React
// import React from "react";
// // Testing
// import { render, screen } from "@testing-library/react";
// import { composeStories } from "@storybook/testing-react";
// // Stories
// import * as stories from "./index.stories";
//
// const { Default } = composeStories(stories);
//
// describe("SignUp", () => {
//   it("renders and displays a button", () => {
//     render(<Default />);
//     expect(getByRole("testId/signUpButton"));
//   });
// });
