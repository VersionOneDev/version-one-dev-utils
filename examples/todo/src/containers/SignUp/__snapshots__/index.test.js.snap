// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp Matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <form
        data-testid="ref"
        novalidate=""
      >
        <p>
          First Name
        </p>
        <input
          class="text-black"
          name="firstName"
          placeholder="Enter first name"
          value=""
        />
        <span
          class="text-green-500"
        >
          Valid
        </span>
        <br />
        <p>
          Surname
        </p>
        <input
          class="text-black"
          name="surname"
          placeholder="Enter surname"
          value=""
        />
        <span
          class="text-red-500"
        />
        <br />
        <p>
          Age
        </p>
        <input
          class="text-black"
          name="age"
          placeholder="Enter age"
          value=""
        />
        <span
          class="text-red-500"
        />
        <br />
        <p>
          Email
        </p>
        <input
          class="text-black"
          name="email"
          placeholder="Enter email address"
          value=""
        />
        <span
          class="text-red-500"
        />
        <br />
        <p>
          Password
        </p>
        <input
          class="text-black"
          name="password"
          placeholder="Enter password"
          value=""
        />
        <span
          class="text-red-500"
        />
        <br />
        <button
          class="text-green-500"
          data-testid="ref/signUpButton"
        >
          Enter Details
        </button>
      </form>
    </div>
  </body>,
  "container": <div>
    <form
      data-testid="ref"
      novalidate=""
    >
      <p>
        First Name
      </p>
      <input
        class="text-black"
        name="firstName"
        placeholder="Enter first name"
        value=""
      />
      <span
        class="text-green-500"
      >
        Valid
      </span>
      <br />
      <p>
        Surname
      </p>
      <input
        class="text-black"
        name="surname"
        placeholder="Enter surname"
        value=""
      />
      <span
        class="text-red-500"
      />
      <br />
      <p>
        Age
      </p>
      <input
        class="text-black"
        name="age"
        placeholder="Enter age"
        value=""
      />
      <span
        class="text-red-500"
      />
      <br />
      <p>
        Email
      </p>
      <input
        class="text-black"
        name="email"
        placeholder="Enter email address"
        value=""
      />
      <span
        class="text-red-500"
      />
      <br />
      <p>
        Password
      </p>
      <input
        class="text-black"
        name="password"
        placeholder="Enter password"
        value=""
      />
      <span
        class="text-red-500"
      />
      <br />
      <button
        class="text-green-500"
        data-testid="ref/signUpButton"
      >
        Enter Details
      </button>
    </form>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`UserAvatar Matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <form
        data-testid="ref"
        novalidate=""
      >
        <p>
          First Name
        </p>
        <input
          class="text-black"
          name="firstName"
          placeholder="Enter first name"
          value=""
        />
        <span
          class="text-green-500"
        >
          Valid
        </span>
        <br />
        <p>
          Surname
        </p>
        <input
          class="text-black"
          name="surname"
          placeholder="Enter surname"
          value=""
        />
        <span
          class="text-red-500"
        />
        <br />
        <p>
          Age
        </p>
        <input
          class="text-black"
          name="age"
          placeholder="Enter age"
          value=""
        />
        <span
          class="text-red-500"
        />
        <br />
        <p>
          Email
        </p>
        <input
          class="text-black"
          name="email"
          placeholder="Enter email address"
          value=""
        />
        <span
          class="text-red-500"
        />
        <br />
        <p>
          Password
        </p>
        <input
          class="text-black"
          name="password"
          placeholder="Enter password"
          value=""
        />
        <span
          class="text-red-500"
        />
        <br />
        <button
          class="text-green-500"
        >
          Enter Details
        </button>
      </form>
    </div>
  </body>,
  "container": <div>
    <form
      data-testid="ref"
      novalidate=""
    >
      <p>
        First Name
      </p>
      <input
        class="text-black"
        name="firstName"
        placeholder="Enter first name"
        value=""
      />
      <span
        class="text-green-500"
      >
        Valid
      </span>
      <br />
      <p>
        Surname
      </p>
      <input
        class="text-black"
        name="surname"
        placeholder="Enter surname"
        value=""
      />
      <span
        class="text-red-500"
      />
      <br />
      <p>
        Age
      </p>
      <input
        class="text-black"
        name="age"
        placeholder="Enter age"
        value=""
      />
      <span
        class="text-red-500"
      />
      <br />
      <p>
        Email
      </p>
      <input
        class="text-black"
        name="email"
        placeholder="Enter email address"
        value=""
      />
      <span
        class="text-red-500"
      />
      <br />
      <p>
        Password
      </p>
      <input
        class="text-black"
        name="password"
        placeholder="Enter password"
        value=""
      />
      <span
        class="text-red-500"
      />
      <br />
      <button
        class="text-green-500"
      >
        Enter Details
      </button>
    </form>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
